CI: Add automatic PR creation for source string updates (#4544)
Cockatrice/Cockatrice: .github/CONTRIBUTING.md

If you're about to propose a change that adds or modifies any translatable
string in the code, you don't need to take care of adding the new strings to
the translation files.<br>

> We have an automted process setup to update our language source files accordingly

Reviewer: ```suggestion
We have an automated process to update our language source files on a schedule
```

and provide the translators on Transifex with the new contents.<br>
Maintainers can also manually trigger this on demand.

### Maintaining Translations (for maintainers) ###

When new translatable strings have been added to the code, a maintainer has to
make them available to translators on Transifex.

> To help with that, we have an automated CI run setup, that regularly looks at the

Reviewer: github calls them workflows instead of CI run setup and I'd like to keep this a bit more consistent

Author: Yep. That makes sense.

`setup` should have been `set up`. Will just remove it.
